### PR TITLE
feat(status): #CRRE-407 check every value of lde commands

### PR DESCRIPTION
**Pull request by sjollois**
## Describe your changes
Check every value of lde commands
## Checklist tests
check if status update
## Issue ticket number and link
[https://jira.support-ent.fr/browse/CRRE-407](url)
## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/java/fr/openent/crre/cron/updateStatus.java [hunk 1]
```diff
@@ -60,9 +61,10 @@ private void updateCommand(Scanner sc, int part, final Handler<Either<String, Js
                 String userLine = sc.nextLine();
                 String[] values = userLine.split(Pattern.quote("|"));
                 JsonObject order = new JsonObject();
-                order.put(Field.STATUS, values[21]);
-                order.put("id", values[22]);
-                if (!values[22].equals("0")) {
+                if (!values[21].isEmpty() && NumberUtils.isParsable(values[21]) &&
```
Comment (JinDouang) [inline src/main/java/fr/openent/crre/cron/updateStatus.java:64]: Vu avec @sjollois pour faire un enum qui "définirait" de 1 à 22 le titre pour chaque colonne

il pourra faire `values[TYPE_DOCUMENT_CSV.STATUS]`  (21 = status)

File: src/main/java/fr/openent/crre/cron/updateStatus.java [hunk 2]
```diff
@@ -60,9 +61,10 @@ private void updateCommand(Scanner sc, int part, final Handler<Either<String, Js
...
+                        !values[22].isEmpty() && NumberUtils.isParsable(values[22]) && !values[22].equals("0")) {
```
Comment (JinDouang) [inline src/main/java/fr/openent/crre/cron/updateStatus.java:65]: Idem pour 22 = id

File: src/main/java/fr/openent/crre/cron/updateStatus.java (hunk 1 above)
Comment (JinDouang) [reply src/main/java/fr/openent/crre/cron/updateStatus.java:64]: A vérifier aussi si ce n'est pas null

**Review by JinDouang — CHANGES_REQUESTED** [review]

File: src/main/java/fr/openent/crre/cron/updateStatus.java (hunk 1 above)
Comment (sjollois) [reply src/main/java/fr/openent/crre/cron/updateStatus.java:64]: la vérif du null n'est pas utile car NumberUtils.isParsable le vérifie déjà

**Review by JinDouang — APPROVED** [review]

**Pull request merged**